### PR TITLE
Great progress

### PR DESCRIPTION
**Pull request by wattiii**
Hey I added a README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Explain here what the project is all about.
```
Comment (axecopfire) [inline README.md]: Can you add the title of this project using a
# Header

Comment (wattiii) [reply README.md]: With HTML markings?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Explain here what the project is all about. 
+This project should allow for a place where python code can be comoditized. 
+It also allows me to learn fundamental coding principles, and build a portfolio for others to see.
```
Comment (axecopfire) [inline README.md]: This looks like a bulled list to me. Can you add bullets please?

* fundamental coding principles
* build a portfolio

File: README.md (hunk 1 above)
Comment (axecopfire) [reply README.md]: With a markdown `#`

Comment (andrew-t-james) [inline README.md]:
```suggestion
# Explain here what the project is all about. 
```

File: README.md (hunk 2 above)
Comment (wattiii) [reply README.md]: I did this.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# README File
++ This file explains what the project is all about. 
++ This project should allow for a place where python code can be comoditized. 
++ It also allows me to learn fundamental coding principles, and build a portfolio for others to see.
```
Comment (axecopfire) [inline README.md]:
```suggestion
+ It also allows me to learn:
   - fundamental coding principles
   -  portfolio building
```

**Review by axecopfire — APPROVED** [review]

**Pull request merged**